src/site/content/en/fast/debugging-layout-shifts/index.md

```diff
@@ -0,0 +1,457 @@
+    thread is idle to report on layout shifts. As a result, depending on how
+    busy the main thread is, there may be a slight delay between when a layout
+    shift occurs and when it is logged in the console.
+*   This script ignores layout shifts that occurred within 500ms of user input
+    and therefore do not count towards CLS.
+
+
+
+Information about layout shifts is reported using a combination of two APIs: the
+[`LayoutShift`](https://wicg.github.io/layout-instability/#layoutshift) and
+[`LayoutShiftAttribution`](https://wicg.github.io/layout-instability/#sec-layout-shift-attribution)
+interfaces. Each of these interfaces are explained in more detail in the
+following sections.
+
+
+#### LayoutShift
+
+Each layout shift is reported using the `LayoutShift` interface. The contents of
+an entry look like this:
+
+```javascript
+duration: 0
+entryType: "layout-shift"
+hadRecentInput: false
+lastInputTime: 0
+name: ""
+sources: (3) [LayoutShiftAttribution, LayoutShiftAttribution, LayoutShiftAttribution]
+startTime: 11317.934999999125
+value: 0.17508567530168798
+```
+
+The entry above indicates a layout shift during which three DOM elements changed
+position. The layout shift score of this particular layout shift was `0.175`.
+
+These are the properties of a `LayoutShift` instance that are most relevant to
+debugging layout shifts:
+
+
+
+*   `sources`: The `sources` property lists the DOM elements that moved during
+    the layout shift. This array can contain up to five sources. In the event
+    that there are more than five elements impacted by the layout shift, the
+    five largest (as measured by impact on layout stability) sources of layout
+    shift are reported. This information is reported using the LayoutShift
+    Attribution interface (explained in more detail below).
+*   `value`: The `value` property reports the [layout shift
+    score](https://web.dev/cls/#layout-shift-score) for a particular layout
+    shift.
+*   `hadRecentInput`: The `hadRecentInput` property indicates whether a layout
+    shift occurred within 500 milliseconds of user input.
+*   `startTime`: The `startTime` property indicates when a layout shift
+    occurred. `startTime` is indicated in milliseconds and is measured relative
+    to the [time that the page load was
+    initiated](https://www.w3.org/TR/hr-time-2/#sec-time-origin).
+*   `duration`: The `duration` property will always be set to `0`. This property
+    is inherited from the
+    [`PerformanceEntry`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceEntry)
+    interface (the `LayoutShift` interface extends the `PerformanceEntry`
+    interface). However, the concept of duration does not apply to layout shift
+    events, so it is set to `0`. For information on the `PerformanceEntry`
+    interface, refer to the
+    [spec](https://w3c.github.io/performance-timeline/#the-performanceentry-interface).
```

Comment:
I think this might be easier to parse as a table (with two columns, property and description).